src/main/java/com/direwolf20/buildinggadgets/common/building/placement/SetBackedBuildView.java

```diff
@@ -0,0 +1,61 @@
+package com.direwolf20.buildinggadgets.common.building.placement;
+
+import com.direwolf20.buildinggadgets.common.building.PlacementTarget;
+import com.direwolf20.buildinggadgets.common.building.Region;
+import com.direwolf20.buildinggadgets.common.building.view.IBuildContext;
+import com.direwolf20.buildinggadgets.common.building.view.IBuildView;
+import com.direwolf20.buildinggadgets.common.util.spliterator.MappingSpliterator;
+import net.minecraft.util.math.BlockPos;
+
+import java.util.Set;
+import java.util.Spliterator;
+
+public class SetBackedBuildView implements IBuildView {
```

Comment:
Might actually be unusued, would need to do a usage search. It used to back InvertedPlacementEvaluator with the unsorted placementTargets.

It's just a BuildView with a set of placementTargets behind it. I honestly don't know what I could write as doc for that, because in my opinionnthe name already says absolutly everyrhing.

Reply:
If unused or highly unneeded the remove